CHANGELOG.md

```
@@ -1,5 +1,19 @@
 # CHANGELOG
 
+## v11.2.0
+
+*Jan 23, 2026*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)